lib/action_controller/serialization.rb

```diff
@@ -31,6 +31,7 @@ def get_serializer(resource, options = {})
         serializable_resource.serialization_scope_name = _serialization_scope
         begin
           serializable_resource.adapter
+          serializable_resource
```

Comment:
Take a look on this commit message https://github.com/maurogeorge/active_model_serializers/commit/981c2293f4ca46daf1ebcd51468497611a389687